.lando.yml

```diff
@@ -15,6 +15,8 @@ services:
       - update-ca-certificates -f -v
     run_as_root:
       - /bin/sh -c "if [ ! -d /var/www/.terminus/plugins/terminus-secrets-plugin ]; then { mkdir -p /var/www/.terminus/plugins; composer create-project -d /var/www/.terminus/plugins pantheon-systems/terminus-secrets-plugin:~1; chown -R www-data:www-data /var/www/.terminus; } fi"
+    build:
```

Comment:
In case you are only targeting your own local env, you can move these settings into ".lando.local.yml" which is ignored by Git but still loaded by `lando rebuild`

Reply:
great idea, I'll move it over.